core/DataTable/Filter/CalculateEvolutionFilter.php

```diff
@@ -191,9 +191,9 @@ public static function prependPlusSignToNumber($number)
      */
     private static function getPercentageValue($value, $divisor, $quotientPrecision)
     {
-        if ($value == 0) {
+        if ($value === 0) {
```

Comment:
I guess we can't assume `$value` to be an integer. Could also be a `float`.
```suggestion
        if ((float) $value === 0.0) {
```

Note: Casting to integer instead would be incorrect as `(int) 0.25` would result in `0`